client/apollo/react/src/ProgressBarGroup/ProgressBarGroupCommon.tsx

```diff
@@ -42,10 +44,10 @@ export const ProgressBarGroup = ({
       className={classNames("af-progress-bar-group", className)}
     >
       {[...Array(nbSteps).keys()].map((index) => (
-        <ProgressBarComponent
+        <ItemProgressBarComponent
           key={`${stepperId}-${index}`}
           value={getCurrentProgress(index)}
-          active={index === currentStep}
+          aria-current={index === currentStep}
```

Comment:
```suggestion
          aria-current={index === currentStep ? true : undifined}
```

aria less is best ...

Reply:
Yes, but i will use "und**E**fined" i think 😆